src/poliastro/czml/extract_czml.py

```diff
@@ -129,10 +175,13 @@ def _change_custom_params(self, ellipsoid, pr_map, scene3D):
 
         if ellipsoid is None:
             ellipsoid = [6378137.0, 6378137.0, 6356752.3142451793]
```

Comment:
Now that we have these values in constants, before making this general for any ellipsoid, can we at least take the values from there?

Reply:
Sure but I'm not sure whether I want to replace them with Earth's constants since those are based on Cesium's default Ellipsoid ``WGS84`` values (perhaps we could also introduce it as a constant)

Reply:
It's a valid concern. The way I see this is:

a) The discrepancies in radius is an Earth-specific problem
b) CZML is a visualization system, so if I have to choose I prefer the propagation to have correct results

So, even though what we are doing is not exactly accurate, it is up to ~30 meters.